eZ/Publish/Core/Repository/Values/ObjectState/ObjectState.php

```diff
@@ -26,16 +23,33 @@
  */
 class ObjectState extends APIObjectState
 {
-    use MultiLanguageTrait;
-    use MultiLanguageNameTrait;
-    use MultiLanguageDescriptionTrait;
```

Comment:
what about using a *(internal documented)* constant that can be set to the fallback language propert, _main_ (defualt), _inital_ (versionInfo, but not applicable here) and _default_ in this case